_data/agencies.yml

```diff
@@ -42,6 +42,11 @@
   acronym: DOJ
   logo: assets/img/logos/agencies/USDOJ.png
   featured: true
+-
+  name: Department of Labor
+  acronym: DOL
+  logo: assets/img/logos/agencies/USDOL.svg
+  featured: true
```

Comment:
`featured: true` is to show a subset of agencies on the homepage. I don't see anything about DOL being on the homepage, so just delete L49 line and we'll be set!